test/api/contenttype.spec.ts

```diff
@@ -46,6 +46,19 @@ describe('ContentType Query API test cases', () => {
       expect(query.entries[0].created_at).toBeDefined();
     }
   });
+
+  it('should get entries which does not match the fieldUid - notExists', async () => {
+  console.log("🚀 ~ makeContentType ~ stack:", await stack.ContentType().find())
```

Comment:
@harshithad0703 Is it required for printing response?

Reply:
removed the console